test.js

```diff
@@ -16,19 +16,29 @@ describe('zipCodeApi', function () {
     beforeEach(function () {
       zipCodeApi.init({
         // Can use real key and comment out mock to test actual API
-        apiKey: 'fake-key',
-      });
+        // apiKey: 'fake-key',
+        // clientKey: 'fake-key',
+        // domain: 'www.zipcodeapi.com',
 
-      mock = fetchMock.mock('https://www.zipcodeapi.com/rest/fake-key/info.json/90210/radians', {
-        city: 'Beverly Hills',
-        state: 'CA',
+        apiKey: 'ADD_REAL_KEY',
+        clientKey: 'ADD_REAL_KEY',
+        domain: 'www.zipcodeapi.com',
+        // domain: 'localhost',
+        // domain: '162.245.142.87',
```

Comment:
I've tried the above domains testing locally but I always get an unauthorized error. Note you need to change apiKey and clientKey to the actual keys (found in doc below) before running `yarn test`.

https://www.zipcodeapi.com/ClientKeySetup/8804